### PR TITLE
MGDOBR-527 - Report Portal integration for Junit 5 tests

### PR DESCRIPTION
**Pull request by jiripetrlik**
[MGDOBR-527](https://issues.redhat.com/browse/MGDOBR-527)

I followed the instructions written here: https://github.com/reportportal/agent-java-junit5 . Integration for Cucumber tests will follow.

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [*] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [*] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [*] All new functionality is tested
- [*] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [*] Pull Request contains link to the JIRA issue
- [*] Pull Request contains link to any dependent or related Pull Request
- [*] Pull Request contains description of the issue
- [*] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [*] Check if code and Github action workflows have been modified and if the modification is safe
- [*] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Review by jomarko — COMMENTED** [review]
Could we please explain the presence/content of `reporportal.properties` in some README?

**Comment by jiripetrlik** [comment]
@jomarko 
Good point. I'm thinking where to put this info as there is no single module for Report Portal. Will probably create something like REPORT_PORTAL.md in the root directory. What do you think?

**Review by jomarko — APPROVED** [review]
Thank you @jiripetrlik

File: pom.xml [hunk 1]
```diff
@@ -27,6 +27,14 @@
...
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- Report portal properties -->
+    <rp.endpoint>http://127.0.0.1:8080</rp.endpoint>
+    <rp.uuid>8c5b8aae-48ea-4984-a229-4ca8c7bfc0e9</rp.uuid>
+    <rp.launch>sandbox</rp.launch>
+    <rp.project>default_personal</rp.project>
+    <rp.attributes>type:dummy</rp.attributes>
+    <rp.enable>false</rp.enable>
```
Comment (sutaakar) [inline pom.xml:37]: Is this needed?

Comment (jiripetrlik) [reply pom.xml:37]: @sutaakar 
I think yes at least rp.enable is necessary to make sure that we are sending the data only when needed. Others provide some default values.

**Review by sutaakar — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (sutaakar) [reply pom.xml:37]: ok

**Pull request merged**